lib/zendesk_apps_support/assets/src.js.erb

```diff
@@ -34,4 +34,4 @@ var app = ZendeskApps.defineApp(<%= iframe_only ? 'null' : 'source' %>)
     frameworkVersion: <%= framework_version.to_json %>
   });
 
-ZendeskApps[<%= name.to_json %>] = app;
+ZendeskApps[<%= id.to_json %>] = app;
```

Comment:
This might break ZAM test, for example this one https://github.com/zendesk/zendesk_app_market/blob/master/spec/features/builder_spec.rb#L469. cc @mtsolakiszen

Reply:
Yes, I can see this change break ZAM/ZAT tests.

Reply:
Before merging this PR, should we have PRs to update those tests ready to be merged as well?